js/libs/keycloak-js/pom.xml

```diff
@@ -73,4 +73,63 @@
             </plugin>
         </plugins>
     </build>
+    <profiles>
```

Comment:
This looks unnecessarily complex, and adds some custom Maven registry settings just for this module. Why not just attach the tgz with:

```
            <plugin>
                <groupId>org.codehaus.mojo</groupId>
                <artifactId>build-helper-maven-plugin</artifactId>
                <executions>
                    <execution>
                        <id>attach-artifacts</id>
                        <phase>package</phase>
                        <goals>
                            <goal>attach-artifact</goal>
                        </goals>
                        <configuration>
                            <artifacts>
                                <artifact>${project.basedir}/assembly.xml
                                    <file>target/keycloak-js-${project.version}.tgz</file>
                                    <type>tar.gz</type>
                                </artifact>
                            </artifacts>
                        </configuration>
                    </execution>
                </executions>
            </plugin>
```

and change the configuration for `maven-deploy-plugin` to `true. That way it's just uploaded along the other artifacts.

Reply:
I can certainly do this. I just didn't want  to alter original maven build behavior.